### PR TITLE
Adding tensornetwork to setup.py

### PR DESCRIPTION
**Pull request by antalszava**
**Context:**
The Tensornetwork is missing as a requirement in `setup.py` and this causes tests of plugins (e.g. PennyLane-Qiskit) to fail when using a device from PennyLane core.

**Description of the Change:**
Adding the Tensornetwork library as a requirement in `setup.py`

**Benefits:**
Fixing Travis tests for plugins.

**Possible Drawbacks:**
N/A

**Related GitHub Issues:**
N/A

### CODE REVIEW

**Comment by antalszava** [comment]
Hi Josh, yes, the file was originally created with CRLF (Windows) line breaks, modified them to LF (Unix based).

Push the pinned version. Thanks!

**Comment by josh146** [comment]
Note that the `expt.tensornet` device only works with v0.1.1, not version 0.0.7. So we will need to come up with a better fix. In particular:

* Update `expt.tensornet` to work with the latest version, v0.2.0

* Work out what to do with Python 3.5 support.

File: setup.py [hunk 1]
```diff
@@ -1,81 +1,81 @@
...
+                    'pennylane.templates',
+                    'pennylane.plugins',
+                    'pennylane.optimize',
+                    'pennylane.interfaces'
+                ],
+    'entry_points': {
+        'pennylane.plugins': [
+            'default.qubit = pennylane.plugins:DefaultQubit',
+            'default.gaussian = pennylane.plugins:DefaultGaussian',
+            'expt.tensornet = pennylane.plugins.expt_tensornet:TensorNetwork'
```
Comment (co9olguy) [inline setup.py]: This is just to prevent `TensorNetwork` import in `__init__.py` from causing an error if an unrelated plugin is used, correct?

**Comment by co9olguy** [comment]
> Note that the `expt.tensornet` device only works with v0.1.1, not version 0.0.7. So we will need to come up with a better fix. In particular:
> 
>     * Update `expt.tensornet` to work with the latest version, v0.2.0
> 
>     * Work out what to do with Python 3.5 support.

Where does v0.0.7 play a role? I think I developed on v0.1.1

File: setup.py (hunk 1 above)
Comment (josh146) [reply setup.py]: Right. This also avoids changing setup.py requirements until it's ready for release.

To be honest, there isn't really any reason to import the devices into `plugins/__init__.py`. By keeping them contained to their own module, import errors specific to a device will only occur on explicit device load.

**Comment by josh146** [comment]
> Where does v0.0.7 play a role? I think I developed on v0.1.1

If you pip install tensornetwork on Python 3.5, it pulls in v0.0.7. This is just a slight inconvenience for now, and we can work out how best to package this later.

**Pull request merged**